boards/feather-m0/doc.txt

```diff
@@ -65,6 +65,21 @@ and define the required WiFi parameters, for example:
 For detailed information about the parameters, see section
 @ref drivers_atwinc15x0.
 
+### Using with LoRa module
+
+To enable the LoRa module available on the
+[Feather M0 LoRa](https://learn.adafruit.com/adafruit-feather-m0-radio-with-lora-radio-module)
+variant of the board and automatically be compatible with LoRa applications
+add the `feather-m0-lora` to your project `Makefile`.
+
+```
+BOARD = feather-m0-lora
+USEMODULE += netdev_default
```

Comment:
Let's make this reflective of the WiFi doc as well:

~~~suggestion
To enable the LoRa module available on the
[Feather M0 LoRa](https://learn.adafruit.com/adafruit-feather-m0-radio-with-lora-radio-module)
variant of the board automatically for LoRa applications,
use `feather-m0-lora` as board:

```
make BOARD=feather-m0-lora -C examples/gnrc_lorawan
~~~

Reply:
@leandrolanzieri could you confirm, that there are no further parameters needed, e.g. keys and such for LoRa?

Reply:
I'd say that that other settings (such as keys) are out of the scope of the documentation of the board. Many of these parameters depend on the stack being used (GNRC LoRaWAN or Semtech LoraMac). I'd expect the user to check the application README to check how to configure these parameters. So it looks OK to me as proposed.